dev-cpp/clucene/files/clucene-2.3.3.4-fix-binary-function.patch

```diff
@@ -0,0 +1,153 @@
+diff --git a/src/core/CLucene/index/_Term.h b/src/core/CLucene/index/_Term.h
```

Comment:
1. Please run iwdevtools' `scrub-patch` on this patch.
2. At the top of the patch (any patch, really), include both a description of what it's doing & why, and a link to any relevant Gentoo bugs + upstream bugs + ...